libmscore/pitchspelling.h

```diff
@@ -25,6 +25,7 @@ enum class Key;
 const int   INVALID_PITCH      = -1;
 
 // a list of tpc's, with legal ranges, not really an enum, so no way to cnvert into a class
+// DO NOT rearrange this enum, otherwise the logic in scale.cpp will break!!!
 enum Tpc : signed char {
```

Comment:
This comment seems a bit out of place: a lot of things would break if this enum is mindlessly changed!!! The order of the TPC's is independent from MuseScore and pervades the whole Western music theory...